lib/osut/utils.rb

```diff
@@ -34,14 +34,14 @@ module OSut
   # DEBUG for devs; WARN/ERROR for users (bad OS input), see OSlg
   extend OSlg
 
-  TOL  = 0.01         # default distance tolerance (m)
-  TOL2 = TOL * TOL    # default area tolerance (m2)
-  DBG  = OSlg::DEBUG  # see github.com/rd2/oslg
-  INF  = OSlg::INFO   # see github.com/rd2/oslg
-  WRN  = OSlg::WARN   # see github.com/rd2/oslg
-  ERR  = OSlg::ERROR  # see github.com/rd2/oslg
-  FTL  = OSlg::FATAL  # see github.com/rd2/oslg
-  NS   = "nameString" # OpenStudio object identifier method
+  TOL  = 0.01             # default distance tolerance (m)
+  TOL2 = TOL * TOL        # default area tolerance (m2)
+  DBG  = OSlg::DEBUG.dup  # see github.com/rd2/oslg
+  INF  = OSlg::INFO.dup   # see github.com/rd2/oslg
+  WRN  = OSlg::WARN.dup   # see github.com/rd2/oslg
+  ERR  = OSlg::ERROR.dup  # see github.com/rd2/oslg
+  FTL  = OSlg::FATAL.dup  # see github.com/rd2/oslg
```

Comment:
This is an attempt to circumvent (often numerous) Ruby warnings related to duplication of constants. These pop up when building/using parent applications relying on gems (like _TBD_), which have _OSut_ as a dependency. That's possibly 3 to 4 gems harmonizing constants, while trying to keep things concise (e.g. not having to drag module namespaces extended). It will take a while to update/build/test larger, multi-gem applications before making a definitive decision.